apInitPassthru.py

```diff
@@ -323,6 +343,7 @@ def mlrs_open_passthrough(comport, baudrate, serialx, options=''):
     parser.add_argument("-findport", action='store_true', help = 'Find port')
     parser.add_argument("-findbaud", action='store_true', help = 'Find baudrate')
     parser.add_argument("-nosysboot", action='store_true', help = 'Do not put into system boot')
+    parser.add_argument("-scripting", action='store_true', help = 'Set scripting protocol first')
```

Comment:
nitpick, an option "scripting" is a bit confusing, would wish there would be a better option name

Reply:
If you want, choose one please.